core/View.php

```diff
@@ -93,11 +95,7 @@ public function setView($requestedView = false, $renderName = false)
                 if (!file_exists($view)) {
                     throw new \Exception('View: "' . $view . '" could not be found.');
                 }
-                if($renderName !== false) {
-                    $this->$renderName = $this->process($view);
-                } else {
-                    $this->view = $this->process($view);
-                }
+                $this->view = $view;
             } else {
                 throw new \Exception('Please provide a view for setView.');
```

Comment:
Why do you even allow this method to be called without a requestedView variable?  Seems like you could remove a lot of code if you simply didn't set $requestedView = false as an optional argument.  Seeing as how it's not actually optional.